src/components.ts

```diff
@@ -28,14 +29,15 @@ export type MessageComponent =
 	| RoleSelect
 	| MentionableSelect
 	| ChannelSelect
-	| InputText
```

Comment:
So .... this would be a typescript breaking change right?  I don't think we wanna do that in a minor